client/src/common/utils/helpers.js

```diff
@@ -0,0 +1,15 @@
+export const areStringsEqual = (string1, string2) =>
+  string1.toUpperCase().trim() === string2.toUpperCase().trim();
+
+export const isUrlValid = string => {
+  const pattern = new RegExp(
+    '^(https?:\\/\\/)?' +
+      '((([a-z\\d]([a-z\\d-]*[a-z\\d])*)\\.)+[a-z]{2,}|' +
+      '((\\d{1,3}\\.){3}\\d{1,3}))' +
+      '(\\:\\d+)?(\\/[-a-z\\d%_.~+]*)*' +
+      '(\\?[;&a-z\\d%_.~+=-]*)?' +
+      '(\\#[-a-z\\d_]*)?$',
+    'i'
+  );
+  return !!pattern.test(string);
```

Comment:
? doesn't `test` method returns boolean already? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/test

Reply:
yes, it returns, but the `isUrlValid` function is surrounded by braces so I need to type return here

Reply:
remove `!!`